Duckling/AmountOfMoney/KA/Rules.hs

```diff
@@ -0,0 +1,399 @@
+import Duckling.Types
+import qualified Duckling.AmountOfMoney.Types as TAmountOfMoney
+import qualified Duckling.Numeral.Types as TNumeral
+
+ruleUnitAmount :: Rule
+ruleUnitAmount = Rule
+  { name = "<unit> <amount>"
+  , pattern =
+    [ Predicate isCurrencyOnly
+    , Predicate isPositive
+    ]
+  , prod = \case
+      (Token AmountOfMoney AmountOfMoneyData{TAmountOfMoney.currency = c}:
+       Token Numeral NumeralData{TNumeral.value = v}:
+       _) -> Just . Token AmountOfMoney . withValue v $ currencyOnly c
+      _ -> Nothing
+  }
+
+rulePounds :: Rule
+rulePounds = Rule
+  { name = "£"
+  , pattern =
+    [ regex "(გირვანქა )?სტერლინგი?ს?(ად)?თ?|(გირვანქა )?ფუნ?ტი?ს?(ად)?თ?"
+    ]
+  , prod = \_ -> Just . Token AmountOfMoney $ currencyOnly Pound
+  }
+
+ruleOtherPounds :: Rule
+ruleOtherPounds = Rule
+  { name = "other pounds"
+  , pattern =
+    [ regex "(ეგვიპტური|ლიბანური) ?ფუნ?ტი?ს?(ად)?თ?"
+    ]
+  , prod = \tokens -> case tokens of
+      (Token RegexMatch (GroupMatch (match:_)):_) -> case Text.toLower match of
+        "ეგვიპტური" -> Just . Token AmountOfMoney $ currencyOnly EGP
+        "ლიბანური" -> Just . Token AmountOfMoney $ currencyOnly LBP
+        _          -> Nothing
+      _ -> Nothing
+  }
+
+ruleRiyals :: Rule
+ruleRiyals = Rule
+  { name = "riyals"
+  , pattern =
+    [ regex "(კატარული|საუდის არაბული) ?რიალი?"
+    ]
+  , prod = \tokens -> case tokens of
+      (Token RegexMatch (GroupMatch (match:_)):_) -> case Text.toLower match of
+        "კატარული" -> Just . Token AmountOfMoney $ currencyOnly QAR
+        "საუდის არაბული"  -> Just . Token AmountOfMoney $ currencyOnly SAR
+        _        -> Nothing
+      _ -> Nothing
+  }
+
+ruleDinars :: Rule
+ruleDinars = Rule
+  { name = "dinars"
+  , pattern =
+    [ regex "(ქუვეითური)? ?დინარი?"
+    ]
+  , prod = \tokens -> case tokens of
+      (Token RegexMatch (GroupMatch (match:_)):_) -> case Text.toLower match of
+        "ქუვეითური" -> Just . Token AmountOfMoney $ currencyOnly KWD
+        _         -> Nothing
+      _ -> Nothing
+  }
+
+ruleDirham :: Rule
+ruleDirham = Rule
+  { name = "დირჰემი"
+  , pattern =
+    [ regex "დირჰემი?"
+    ]
+  , prod = \_ -> Just . Token AmountOfMoney $ currencyOnly AED
+  }
+
+ruleRinggit :: Rule
+ruleRinggit = Rule
+  { name = "ringgit"
+  , pattern =
+    [ regex "(მალაიზი?ური? )?რინგიტი?"
+    ]
+  , prod = \_ -> Just . Token AmountOfMoney $ currencyOnly MYR
+  }
+
+ruleCent :: Rule
+ruleCent = Rule
+  { name = "cent"
+  , pattern =
+    [ regex "ცენტი?ს?(ად)?თ?|თეთრი?ს?(ად)?თ?|პენსი?ს?(ად)?თ?|ევროცენტი?ს?(ად)?თ?"
+    ]
+  , prod = \_ -> Just . Token AmountOfMoney $ currencyOnly Cent
+  }
+
+-- ruleTetri :: Rule
+-- ruleTetri = Rule
+--   { name = "tetri"
+--   , pattern =
+--     [ regex "თეთრი?ს?(ად)?თ?"
+--     ]
+--   , prod = \_ -> Just . Token AmountOfMoney $ currencyOnly TETRI
+--   }
+
+ruleLari :: Rule
+ruleLari = Rule
+  { name = "Lari"
+  , pattern =
+    [ regex "ლარი?ს?(ად)?თ?| ?GEL ?"
```

Comment:
try:
```
[ regex "(ლარი?ს?(ად)?თ?)|GEL"
```

BTW `GEL` should be part of the common currency symbol rule in `AmountOfMoney/Rules.hs`.

Reply:
Thanks for your comment, I took it into account and tried to do it, however, when I try to use Georgian symbol for Lari (₾) following error occurs: 

/home/david/duckling4/Duckling/AmountOfMoney/Rules.hs:64:9: error:
    lexical error in string/character literal at character '\8382'